### PR TITLE
update combat pet stats

### PR DESCRIPTION
**Pull request by catgirlseraid**
blaze
eerie 
ender dragon 
enderman
ghoul
golden dragon
golem 
grandma wolf 
griffin 
kuudra 
magma cube
phoenix 
rat 
rift ferret 
skeleton horse
wisp 
skeleton
spider
spirit
tarantula
tiger

### CODE REVIEW

File: items/DROPLET_WISP;1.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   "internalname": "DROPLET_WISP;1",
   "crafttext": "",
   "clickcommand": "viewrecipe",
-  "modver": "2.1.1-PRE",
+  "modver": "2.4.0",
```
Comment (jani270) [inline items/DROPLET_WISP;1.json:46]: lol

File: constants/petnums.json [hunk 2]
```diff
@@ -3907,21 +3903,17 @@
...
       },
       "100": {
-        "otherNums": [
-          10
-        ],
+        "otherNums": [],
         "statNums": {
           "SPEED": 50,
-          "INTELLIGENCE": 2
+          "INTELLIGENCE": -2
```
Comment (jani270) [inline constants/petnums.json:3916]: lets not do that, may break parsing of other projects

File: constants/petnums.json [hunk 3]
```diff
@@ -3907,21 +3903,17 @@
...
     "EPIC": {
       "1": {
-        "otherNums": [
-          10
-        ],
+        "otherNums": [],
         "statNums": {
           "SPEED": 0.5,
-          "INTELLIGENCE": 0.02
+          "INTELLIGENCE": -0.02
```
Comment (jani270) [inline constants/petnums.json:3909]: same here.

**Review by jani270 — CHANGES_REQUESTED** [review]

File: constants/petnums.json (hunk 3 above)
Comment (catgirlseraid) [reply constants/petnums.json:3909]: {ED27BD31-185A-478A-B98C-55DCE1D26804}
should i change the jerry pet too? i didn't do this it was already like that so i just copied the only other pet with negitive stats

Comment (jani270) [reply constants/petnums.json:3909]: Oh i didnt know we already had jerry with that shit, ig its fine

**Review by jani270 — CHANGES_REQUESTED** [review]
surely someone will report it if something is wrong here

**Review by jani270 — APPROVED** [review]
oops wrong button

**Pull request merged**